x/metadata/keeper/msg_server.go

```diff
@@ -178,6 +178,66 @@ func (k msgServer) DeleteScopeOwner(
 	return types.NewMsgDeleteScopeOwnerResponse(), nil
 }
 
+func (k msgServer) UpdateValueOwners(
```

Comment:
Should we be adding the comment for the method name in msg_server.  I know they are missing all over our code base.  I have been trying to be a better about adding them myself (I sometimes forget though).

Reply:
Heh. I haven't been adding them since non of the other ones ever had them, but I prefer having function comments. So I'll add some.

Reply:
Done. I copied the comments from the tx.proto onto their `MsgServer` funcs in here.

I'm guessing that we usually didn't have them since the `msgServer` struct is private and implements the metadata `MsgServer` interface defined automatically from the protos. And that interface has the comments on the functions.

But I like having them in `msg_server.go` too.